pyfolio/perf_attrib.py

```diff
@@ -97,6 +98,37 @@ def perf_attrib(returns, positions, factor_returns, factor_loadings,
             2017-01-01  0.249087  0.935925        1.185012          1.185012
             2017-01-02 -0.003194 -0.400786       -0.403980         -0.403980
     """
+
+    missing_stocks = positions.columns.difference(
+        factor_loadings.index.levels[1]
+    )
+
+    # cash will not be in factor_loadings
+    missing_stocks = missing_stocks.drop('cash')
+
+    if len(missing_stocks) > 0:
```

Comment:
I think it would be good to also compute the average summed allocation (sum holdings over missing stocks and take average over time) of the missing stocks and report that to give a sense of how much will be missing.

Reply:
added this to the missing stocks warning